composer.json

```diff
@@ -106,7 +106,7 @@
         "league/flysystem-sftp-v3": "^3.0",
         "mockery/mockery": "^1.6",
         "nyholm/psr7": "^1.2",
-        "orchestra/testbench-core": "^9.0.15",
+        "orchestra/testbench-core": "^9.1.3",
```

Comment:
This fixed discovering commands via Testbench's Console Kernel (used in tests below)